addons/bestja_frontend_fpbz/templates/pages/footer.xml

```diff
@@ -1,49 +1,49 @@
-                            <div class="col-md-5 mt16 footer_icons_container">
-                                <ul class="col-md-12 footer_icons">
-                                    <li>
-                                        <a class="facebook" href="http://facebook.com"></a>
-                                    </li>
-                                    <li>
-                                        <a class="twitter" href="http://facebook.com"></a>
-                                    </li>
-                                    <li>
-                                        <a class="youtube" href="http://youtube.com"></a>
-                                    </li>
-                                </ul>
-                                <div class="copyrights">
-                                    <p>© 2015,<span t-field="res_company.name"/></p>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </footer>
-            </xpath>
-        </template>
-    </data>
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="new_footer" inherit_id="website.layout" name="New footer">
+            <xpath expr="//footer" position="replace">
+                <footer id="footer_container">
+                    <div class="container-fluid footer_div">
+                        <div class="row clearfix">
+                            <div class="col-md-5 footer_logo_container">
+                                <a class="footer_logo" href="http://www.bankizywnosci.pl/"></a>
+                                <ul class="footer_contact">
+                                    <li class="mt8 mb8">
+                                        <a href="/page/contactus">Kontakt</a>
+                                    </li>
+                                    <li class="mt8 mb8">
+                                        <a href="/page/regulations">Regulamin</a>
+                                    </li>
+                                    <li class="mt8 mb8">
+                                        <a href="http://www.wolontariat.bankizywnosci.pl/">Polityka prywatności</a>
+                                    </li>
+                                    <li class="mt8 mb8">
+                                        <a href="http://www.wolontariat.bankizywnosci.pl/">O nas</a>
+                                    </li>
+                                </ul>                                
+                            </div>
+                            <div class="col-md-2 centered text-center" />
+                            <div class="col-md-5 mt16 footer_icons_container">
+                                <ul class="col-md-12 footer_icons">
+                                    <li>
+                                        <a class="facebook" href="https://www.facebook.com/bankizywnosci?ref=ts"></a>

```

Comment:
We can load it from the database. The same applies to Youtube and Twitter URL – these are saved in Odoo settings.